splitter.go

```diff
@@ -101,14 +117,26 @@ func (sw SplitWriter) WriteTag(name string, value interface{}) error {
 			if !ok {
 				continue // Can't touch that
 			}
-			fieldName = name + "." + fieldName // add prefix
+			if name != "" {
```

Comment:
Interesting.  Under what case would we expect `name` to be the empty string? The parser raises an error if we try to parse the empty string as a tag name so I'd expect this not to happen?

Reply:
This came up because I needed it (and it works).

Imagine I want to read these top-level tags:
```
MY_TAG
ANOTHER_TAG
```

I define a struct: `type struct AllTags {MY_TAG, ANOTHER_TAG float32}`

Then I read it: `device.ReadTag("", &allTags)`

...I think it's ok if the `Locker` doesn't handle this case because I think locking will occur below the `Splitter`.

Reply:
I see, so `name` is like a common prefix for the fields.  I see.